### ISSUE TITLE
Refactor DA outputs

### ISSUE THREAD

**Issue by rajatagarwal-ibm** [terraform-ibm-modules/terraform-ibm-rag-sample-da#45]
**Rename:**

`project_id` -> `watsonx_project_id`
`project_url` -> `watsonx_project_url`
`watsonx_assistant_url` -> `watsonx_assistant_api_url`
`watsonx_discovery_url` -> `watsonx_discovery_api_url`

**Create new outputs:**

* `watsonx_assistant_dashboard_url`
* `watsonx_discovery_dashboard_url`

**Comment by rajatagarwal-ibm** [comment]
**TLDR -** We don't have CRN for assistant and discovery projects which is required for the dashboard URL. One option is to use RestAPI like configure_project did in the SaaS DA. That returns a lot of values (see below), including CRN. 

---
**Assistant Dashboard URL:**

Sample assistant dashboard URL - https://us-south.assistant.watson.cloud.ibm.com/crn%3Av1%3Abluemix%3Apublic%3Aconversation%3Aus-south%3Aa%2Fabac0df06b644a9cabc6e44f55b3880e%3A9bdd9a67-995c-4a82-80ea-edc005a34d4e%3A%3A/assistants/1ed310a1-ace5-4e1a-8a7a-48ce7cb3eaba/home

URL components are:
* Hostname - It's available. Can tweak it based on the region supplied (currently everything is in `us-south` anyway).
* CRN - NOT AVAILABLE. As per the Assistant API docs, there is no provision to return the CRN. See https://cloud.ibm.com/apidocs/assistant-v2#listassistants
* Assistant Project ID: It's available.

**Discovery Dashboard URL:**

Sample discovery dashboard URL - https://us-south.discovery.watson.cloud.ibm.com/v2/instances/crn%3Av1%3Abluemix%3Apublic%3Adiscovery%3Aus-south%3Aa%2Fabac0df06b644a9cabc6e44f55b3880e%3Ab96e09cf-3d44-4a02-80e9-1762c613f1ca%3A%3A/projects

URL components are:
* Hostname - It's available. Can tweak it based on the region supplied (currently everything is in `us-south` anyway).
* CRN - NOT AVAILABLE. As per the Discovery API docs, there is no provision to return the CRN. See https://cloud.ibm.com/apidocs/discovery-data#listprojects

**Note:** Configure project uses the "restapi" provider for set up, but we use scripts. This is the object returned by the Rest API provider

```
crn = {
  "api_data" = tomap({
    "entity" = <REDACTING AS CONTAINS API KEY>
  })
  "api_response" = <REDACTING AS CONTAINS SECRET KEYS>
  "create_method" = "POST"
  "create_path" = "//api.dataplatform.cloud.ibm.com/transactional/v2/projects?verify_unique_name=true"
  "create_response" = "{\"location\":\"/v2/projects/c37a1b40-fb7f-406a-97b7-e27b45dba722\"}"
  "data" = <<-EOT
  {
    "name": "rag-del-test-RAG-sample-project",
    "generator": "watsonx-saas-da",
    "type": "wx",
    "storage": {
      "type": "bmcos_object_storage",
      "guid": "b1a4e4a0-e23e-413d-bc98-b4e894f85827",
      "resource_crn": "crn:v1:bluemix:public:cloud-object-storage:global:a/abac0df06b644a9cabc6e44f55b3880e:b1a4e4a0-e23e-413d-bc98-b4e894f85827::"
    },
    "description": "WatsonX AI project for RAG pattern sample app",
    "public": true,
    "tags": ["watsonx-ai-SaaS","RAG-sample-project"],
    "compute": [
      {
        "name": "rag-del-watson-machine-learning-instance",
        "guid": "85dd3375-b94a-4d02-abeb-867d1ca5b376",
        "type": "machine_learning",
        "crn": "crn:v1:bluemix:public:pm-20:us-south:a/abac0df06b644a9cabc6e44f55b3880e:85dd3375-b94a-4d02-abeb-867d1ca5b376::"
      }
    ]
  }
  
  EOT
  "debug" = tobool(null)
  "destroy_data" = tostring(null)
  "destroy_method" = "DELETE"
  "destroy_path" = "//api.dataplatform.cloud.ibm.com/transactional{id}"
  "force_new" = tolist(null) /* of string */
  "id" = "/v2/projects/c37a1b40-fb7f-406a-97b7-e27b45dba722"
  "id_attribute" = "location"
  "object_id" = tostring(null)
  "path" = "//api.dataplatform.cloud.ibm.com"
  "query_string" = tostring(null)
  "read_data" = tostring(null)
  "read_method" = "GET"
  "read_path" = "//api.dataplatform.cloud.ibm.com{id}"
  "read_search" = tomap(null) /* of string */
  "update_data" = <<-EOT
  {
    "name": "rag-del-test-RAG-sample-project",
    "type": "wx",
    "description": "WatsonX AI project for RAG pattern sample app",
    "public": true,
    "compute": [
      {
        "name": "rag-del-watson-machine-learning-instance",
        "guid": "85dd3375-b94a-4d02-abeb-867d1ca5b376",
        "type": "machine_learning",
        "crn": "crn:v1:bluemix:public:pm-20:us-south:a/abac0df06b644a9cabc6e44f55b3880e:85dd3375-b94a-4d02-abeb-867d1ca5b376::",
        "credentials": {}
      }
    ]
  }
  
  EOT
  "update_method" = "PATCH"
  "update_path" = "//api.dataplatform.cloud.ibm.com{id}"
}

```

**Comment by rajatagarwal-ibm** [comment]
Closing as its being already tracked here https://github.com/terraform-ibm-modules/terraform-ibm-rag-sample-da/issues/21
